packages/databricks-vscode/src/bundle/BundleProjectManager.ts

```diff
@@ -171,8 +178,15 @@ export class BundleProjectManager {
     }
 
     public async initNewProject() {
-        const authProvider = await LoginWizard.run(this.cli, this.configModel);
+        let authProvider =
+            this.connectionManager.databricksWorkspace?.authProvider;
+        if (!authProvider) {
+            authProvider = await LoginWizard.run(this.cli, this.configModel);
+        }
```

Comment:
No strong opinions here, but might be nice to ask users if they want to continue using the same auth?